catalog.html

```diff
@@ -0,0 +1,235 @@
+          <a class="navigation-link" href="#">О Седоне</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="catalog.html">Гостиницы</a>
+        </li>
+      </ul>
+      <ul class="navigation-user">
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">
+            <span class="visually-hidden">Поиск.</span>
+          </a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">
+            <span class="visually-hidden">Избранное.</span>
+          </a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">
+            <span class="visually-hidden">Хочу сюда.</span>
+          </a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+
+  <main class="main-inner">
+    <header class="inner-header">
+      <h1 class="inner-title">Гостиницы Судоны</h1>
+      <ul class="breadcrumbs">
+        <li class="breadcrumbs-item">
+          <a class="breadcrumbs-link" href="index.html"><img src="" alt="Дом."></a>
+        </li>
+        <li class="breadcrumbs-item">
+          <a class="breadcrumbs-link">Гостиницы</a>
+        </li>
+      </ul>
+    </header>
+    <section class="catalog-filter-housing">
+      <h2 class="visually-hidden">"Фильтр"</h2>
+      <form class="catalog-filter" action="https://echo.htmlacademy.ru/" method="get">
+        <div class="choice1">
+         <h3 class="catalog-filter-title">Инфраструктура:</h3>
+          <ul class="catalog-filter-list">
+            <li class="catalog-filter-item">
+              <label for="service1">
+                <input id="service1" type="checkbox" name="pool" checked>Бассейн
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label for="service2">
+                <input id="service2" type="checkbox" name="parking" checked>Парковка
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label for="service3">
+                <input id="service3" type="checkbox" name="Wi-Fi">Wi-Fi
+              </label>
+            </li>
+          </ul>
+        </div>
+        <div class="choice2">
+          <h3 class="catalog-filter-title">Тип жилья:</h3>
+          <ul class="catalog-filter-list">
+            <li class="catalog-filter-item">
+              <label for="place1">
+                <input id="place1" type="radio" name="hotel" value="shaving-supplies" checked>Гостиница
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label for="place2">
+                <input id="place2" type="radio" name="motel" value="shaving-supplies">Мотель
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label for="place3">
+                <input id="place3" type="radio" name="apartments" value="shaving-supplies">Апартаменты
```

Comment:
```suggestion
                <input id="place3" type="radio" name="type-of-housing" value="apartments">Апартаменты
```

вот ты поправила 3 коммента. Почему у всех инпут радио я сделала одинаковое имя?

Reply:
т.к. сдесь идет выбор одного элемента из  группы, потому кнопки имеют одно имя на всех.

Reply:
да, правильно